modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSCustomView.testcase

```diff
@@ -191,4 +191,56 @@ definition {
 		}
 	}
 
+	@description = "LPS-164602 Frontend Dataset - column labels are not localized "
+	@priority = "4"
+	test ManagementBarCanBeLocalized {
+		task ("And Given: site URL localized to Spanish //localhost:8080/es/ and when on a customized tab view") {
+			var portalURL = PropsUtil.get("portal.url");
+
+			Navigator.openSpecificURL(url = "${portalURL}/es/frontend-data-set-test-page/");
+		}
+
```

Comment:
Hey @eltonccdantas could you add `Navigator.gotoPage(pageName = "Frontend Data Set Test Page");` here? By [poshi report ](https://testray.liferay.com/reports/production/logs/2022-10/test-1-38/test-portal-acceptance-pullrequest(master)/4534/functional-tomcat90-mysql57-jdk8/0/16/LocalFile.FDSCustomView_ManagementBarCanBeLocalized/index.html.gz?authuser=0) When it changes the language of the page is being directed to  home page and this is breaking test